elastica/rod/factory_function.py

```diff
@@ -192,6 +192,18 @@ def allocate(
     mass[1:] += 0.5 * density * volume
 
     # Set dissipation constant or nu array
+    warnings.warn(
```

Comment:
Please use `logging` module instead of `warnings`. It will become more useful when we want to integrate with configuration modules, like `sacred`.